constants.yml

```diff
@@ -16,6 +16,7 @@ company_summary:
     'private-unlimited' : "Private unlimited company"
     'ltd' : "Private limited Company"
     'plc' : "Public limited Company"
+    'pcc' : "Protected Cell Company"
```

Comment:
Should this be `protected-cell-company` to be consistent with **most** of the rest of this list?